examples/Format_Features/simulation_and_plotting.py

```diff
@@ -0,0 +1,36 @@
+import amici
+import amici.plotting
+import os
+import sys
+import importlib
+import numpy as np
+
+
+def compile_and_simulate_in_AMICI(sbml_name: str):
```

Comment:
```suggestion
def compile_and_simulate_in_amici(sbml_name: str):
```

Lowercase seems to be the Python-standard and the AMICI norm (e.g. except for the not so Python-standard camelCase, `runAmiciSimulations` has a lowercase `amici`).

Reply:
Could add return type